suite-common/wallet-config/src/types.ts

```diff
@@ -54,6 +54,8 @@ export type NetworkFeature =
     | 'coin-definitions'
     | 'nft-definitions';
 
+export type Bip43Path = string; // TODO define a more specific type
```

Comment:
Placeholder for future refactoring - create a more generic string template.
When we relied on inferred types, `NetworkCompatible['bip43Path']` was used, this replaces it.